### DISCUSSION TITLE
RMT GPU hashrate

### DISCUSSION THREAD

**Discussion by old80** [community/community#38293] [Other Feature Feedback, Questions, & Ideas]
Hi, brother, thanks for your great programing jog.
My sugestion would be to taking more advantagem of the GPU power for mining RTM.
Cause several miners are abandoning it, cause of the low profit comparing to the energy cousts.

**Comment by david-engelmann** [comment]
The spike in Hash rates in the post merge ETH  world is making 99% of POW mining not profitable
